app/templates/frameworks/services.html

```diff
@@ -65,91 +64,115 @@ <h2 class="summary-item-heading">{{ framework.name }} is closed for applications
     </p>
   {% endif %}
 
-  {{ summary.heading("Draft services") }}
+  <h2 class="govuk-heading-m">Draft services</h2>
```

Comment:
It'd be good to use [table `<caption>`](https://design-system.service.gov.uk/components/table/#table-captions), but we don't have a clean mechanism to do that AND hide the table if there's no data.

We've been doing this if/else thing in lots of places: https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/afedd090199322b0dfe1545b824e10e03ae7899e/app/templates/buyers/dashboard.html#L33-L39

But I'm concerned about a mix of heading tags and table captions for same-level content if there's multiple tables which might be empty, so I think headings are probably best if they directly precede the table.

Reply:
changed to button in 9011699